styles.css

```diff
@@ -0,0 +1,298 @@
+/* ****Start Mobile first**** */
+
+/* @media screen and (max-width: 500px) { */
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.navbar_section {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  padding-right: 40px;
+  padding-left: 20px;
+  padding-top: 10px;
+  height: 48px;
```

Comment:
- [ ] [OPTIONAL] Great job with your project 💪. Your site overflows kindly fix this 🙏.

![20221202-143146_capture](https://user-images.githubusercontent.com/74814780/205305109-94e861c0-09cb-40c0-a118-9ff9a322b0db.gif)